cyrasterize/glrasterizer.pyx

```diff
@@ -352,13 +356,15 @@ cdef class GLScene:
         glDrawElements(GL_TRIANGLES, self.mesh.trilist.n_vectors * 3,
                 GL_UNSIGNED_INT, <GLvoid*> 0)
 
-        # 3. DETATCH + SWAP BUFFERS
+        # 3. DETACH + SWAP BUFFERS
         # now we're done, can disable the vertex array (for safety)
         glBindVertexArray(0)
 
         cdef GLFWwindow* window = <GLFWwindow*> self.context.window
 
         glfwSwapBuffers(<GLFWwindow *> window)
+        # We should poll for events after buffering but before rendering.
+        glfwPollEvents()
```

Comment:
This is the only inline code change. Which is recommended by the GLFW docs as the window may hang when you change monitors without it.